src/backend/distributed/connection/remote_commands.c

```diff
@@ -25,7 +25,11 @@
 #include "utils/palloc.h"
 
 
-#define MAX_PUT_COPY_DATA_BUFFER_SIZE (8 * 1024 * 1024)
+/*
+ * Setting that controls how many bytes of COPY data libpq is allowed to buffer
+ * internally before we force a flush.
+ */
+int RemoteCopyFlushThreshold = 8 * 1024 * 1024;
```

Comment:
We could probably move this setting to multi_copy.c/h, right after `CopySwitchOverThresholdBytes`?